code/ui/Hud.cs

```diff
@@ -95,13 +100,16 @@ public void CreateHud()
             {
                 if (_panels.Count == 0)
                 {
+                    Effects = Parent.AddChild<Effects>();
```

Comment:
If we remove the earlier `Effects.OnHotReloaded` then we don't have to uniquely handle `Effects` here right?

Reply:
We need this because it's e.g. used in `TTTPerks.OnEquip`